config/crd/kustomization.yaml

```diff
@@ -5,6 +5,7 @@ resources:
 - bases/config.gatekeeper.sh_configs.yaml
 - bases/status.gatekeeper.sh_constraintpodstatuses.yaml
 - bases/status.gatekeeper.sh_constrainttemplatepodstatuses.yaml
+- bases/status.gatekeeper.sh_mutatorpodstatuses.yaml
```

Comment:
this should be in config/overlays/mutation/kustomization.yaml until mutation feature is more stable

Reply:
I think that won't break the tests, I'll confirm before pushing.

Reply:
Having this file here will add it to deploy/gatekeeper.yaml, which is not what we want because this is still experimental.